macosx/Transmission_Prefix.pch

```diff
@@ -1,3 +1,4 @@
 #ifdef __OBJC__
     #import <Cocoa/Cocoa.h>
+    #import "Compatibility.h"
```

Comment:
Do we need this in the precompiled header?

Reply:
I could rename the file to "CocoaCompatibility.h": its specific purpose is to declare the missing declarations from `<Cocoa/Cocoa.h>`.
I believe its best inclusion is here, adjacent to `#import <Cocoa/Cocoa.h>`.